### ISSUE TITLE
Write actual data in the Tutorial:Create a multi-extension FITS (MEF) file from scratch

### ISSUE THREAD

**Issue by janerigby** [astropy/astropy#12161]
I'm trying to learn from the tutorial titled "Create a multi-extension FITS (MEF) file from scratch".  I can see how to make a blank MEF that way.  What I can't figure out is how to populate it with actual data, for example with two numpy arrays.  The tutorial would be much more powerful if it filled the MEF with actual data.  I'd suggest some code, but I can't figure out the MEF part!

https://github.com/astropy/astropy/blob/main/examples/io/create-mef.py

**Comment by janerigby** [comment]
Also, I just realized that ds9 refuses to read the test.fits file created by the tutorial.  ds9 claims the fits file has an error.

**Comment by pllim** [comment]
Thanks for reporting this. Huh, not sure why the example is writing empty ImageHDU...

**Comment by saimn** [comment]
Right, we could put some fake data there, e.g. `fits.ImageHDU(data=np.arange(5))`.

**Comment by pllim** [comment]
TBH, I don't know why we still have this example gallery here, when there are better ones over at tutorials... Maybe @adrn , @kelle , @eblur et al. can clarify the fate of this example gallery.

**Comment by kelle** [comment]
The plan was to _migrate_ this content to a new home so I think it's worth the effort to improve it.

**Comment by saimn** [comment]
Actually the documentation already explains this better, https://docs.astropy.org/en/latest/io/fits/#creating-a-file-with-multiple-extensions. So I'm convinced by the interest of this tutorial which is very basic.

**Comment by saimn** [comment]
Maybe MEF should be mentioned here instead:
https://learn.astropy.org/rst-tutorials/FITS-images.html#writing-image-data-to-a-fits-file
